app.js

```diff
@@ -190,11 +192,11 @@ fs.readFile(path.resolve(__dirname, 'config/ipbans.txt'), (err, data) => {
 		if (!data[i]) continue;
 		if (data[i].includes('/')) {
 			rangebans.push(data[i]);
-		} else if (!Users.bannedIps[data[i]]) {
-			Users.bannedIps[data[i]] = '#ipban';
+		} else if (!Punishments.bannedIps[data[i]]) {
+			Punishments.bannedIps[data[i]] = '#ipban';
 		}
 	}
-	Users.checkRangeBanned = Cidr.checker(rangebans);
+	Punishments.checkRangeBanned = Cidr.checker(rangebans);
 });
```

Comment:
This was a pretty uncontroversial change?
Why wasn't it done? Do you expect @Zarel's input for everything, @bumbadadabum ?